app/org/sagebionetworks/bridge/models/accounts/ParticipantOptions.java

```diff
@@ -0,0 +1,61 @@
+package org.sagebionetworks.bridge.models.accounts;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+import javax.annotation.Nullable;
+
+import org.sagebionetworks.bridge.dao.ParticipantOption.SharingScope;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Object to deserialize JSON update of participant options. Some or all of the 
+ * values can be missing in the JSON and we will only update values that are non-null.
+ */
+public final class ParticipantOptions {
+
+    private final @Nullable SharingScope sharingScope;
+    private final @Nullable Boolean notifyByEmail;
+    private final @Nullable String externalId;
+    private final @Nullable Set<String> dataGroups;
+    private final @Nullable LinkedHashSet<String> languages;
```

Comment:
We would still need to know for specific properties, if they are coming in as booleans or arrays, and deserialize them correctly into strings. This seems like the easiest way to do it since all the work is left to Jackson.